hq/app/views/iam/iamCredReportBody.scala.html

```diff
@@ -20,8 +20,9 @@
                     <th>Status</th>
                 </tr>
                 @for(cred <- report.humanUsers) {
+
                     <tr>
-                        <td><span>@views.html.fragments.username(cred.username)</span></td>
+                        <td><a target="_blank" href="https://console.aws.amazon.com/iam/home#/users/@{cred.username}">@views.html.fragments.username(cred.username)</a></td>
```

Comment:
We probably can trust AWS, but if we want to be extra careful we could add `rel="noopener noreferrer"` to the anchor tag...

Fun fact:
> The page we're linking to gains partial access to the linking page via the window.opener object.

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
very interesting, 👐   web world is always complex.

Reply:
🤔 Do we want to target blank and change the default behaviour of links? (even speaking as someone who cmd-clicks every link because I love tabs)

If we are going to stick with target blank, maybe we could make the link an icon instead? Other apps use this icon to show that it opens in a new tab:

<img width="220" alt="picture 83" src="https://user-images.githubusercontent.com/8607683/33564879-8f812152-d913-11e7-9c4f-4c0ada41dbac.png">

https://material.io/icons/#ic_open_in_new